### DISCUSSION TITLE
Cannot identify source of background in channels 143 and 144

### DISCUSSION THREAD

**Discussion by whitneyt1** [angelolab/toffy#287] [Q&A] [answered]
Hi! 

We're using toffy to correct some mouse pancreatic islets stained with a smaller panel (attached). We are seeing some strange background in 143 (CD4) and 144 (CD11c), but can't identify the source of the background. 

Attached a screen shot of the CD4 stitched with noodle at 2, 4, 6, 8x in QuPath.
<img width="584" alt="CD4_sourcenoodle_screenshot" src="https://user-images.githubusercontent.com/46573884/201755652-192aacd7-df07-444f-bd06-103c403c44d2.PNG">

We also tried correcting Gold, 143 channel and 144 channel to see if either affected the other, and we did not see noticeable difference. All other channels that affect 143 are empty except 150, which contains GranB. 

[Panel55_MousePanel-toffy.csv](https://github.com/angelolab/toffy/files/10006613/Panel55_MousePanel-toffy.csv)

Our MIBI manager said he noticed a peak in 181 Ta that is similar in peak height to the Au peak, but we are unsure how we'd attempt to correct that out because it is not in the commercial matrix. 

Do you have any advice on identifying the source of the background or correcting it out? We are concerned it will affect CD4 pixel positivity. 

Thanks!

**Comment by ngreenwald** [comment, marked as answer]
When we have a background source we can't identify, we extract all of the masses from 30 to 176 for the FOVs that have the background, and then compare visually to see if any have the same pattern. Once you've identified the source, you can update rosetta to include a new coefficient. This assumes that it's an instrument issue, and not a staining artifact.

**Reply by whitneyt1** [reply]
Thank you! How do you determine the appropriate coefficient value? Thanks!

**Reply by ngreenwald** [reply]
Guess and check. You can look at the total signal in the source contamination channel, the total signal in the target channel, and figure out about what coefficient would work.  Then use the normal process for modifying coefficients to zero in on the one that works
